statefulset-runner/controllers/pdb.go

```diff
@@ -0,0 +1,72 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	policyv1 "k8s.io/api/policy/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const PdbMinAvailableInstances = "50%"
+
+type Updater struct {
```

Comment:
This struct fully qualified name name is `controllers.Updater` which is too broad and ambiguous. Let's rename it to something more specific like for example `controllers.PDBUpdater`?

Reply:
We copied the implementation and related tests from from `eirini-controller`. Your suggestion make sense. I'll do the necessary changes.